ipatests/test_integration/test_installation.py

```diff
@@ -363,6 +364,21 @@ def get_pki_tomcatd_pid(host):
     return(pid)
 
 
+def get_ipa_services_pids(host):
+    ipa_services_name = [
+        "krb5kdc", "kadmin", "named", "httpd", "ipa-custodia",
```

Comment:
ipa-custodia is not defined as a wellknownservice in ipa-4-6 branch. You need to update the definition in ipaplatfrm/base/services.py. Same comment for ipa-dnskeysyncd.